test/e2e/lib/creation.go

```diff
@@ -72,6 +72,16 @@ func (c *Client) CreatePubSubOrFail(pubsub *eventsv1alpha1.CloudPubSubSource) {
 	c.Tracker.AddObj(pubsub)
 }
 
+func (c *Client) CreatePubSubOrFailWithoutClean(pubsub *eventsv1alpha1.CloudPubSubSource) {
```

Comment:
The only difference between `CreatePubSubOrFailWithoutClean` and `CreatePubSubOrFail` is this line https://github.com/capri-xiyue/knative-gcp/blob/ffdbf0872bef3a4e755b01ee3a812a41be4449d2/test/e2e/lib/creation.go#L72
For such smoke tests in this PR, we will delete the sources by ourselves, therefore we don't need the clean